.github/workflows/ci.yml

```diff
@@ -79,7 +80,20 @@ jobs:
                 if: matrix.calculate-code-coverage == 'yes'
                 env:
                     COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                    COVERALLS_PARALLEL: true
+                    COVERALLS_FLAG_NAME: "PHP ${{ matrix.php }} + Symfony@${{ matrix.symfony || '*' }}"
                 run: bin/php-coveralls -v --exclude-no-stmt
+
+    coveralls-finish:
```

Comment:
I would suggest to keep the PR atomic and do only Sf6 support here. For multiupload, we would start a discussion how weird it is to use other coveralls SDK in our SDK client. We should be able to do this action using only our SDK

Reply:
Yes I agree, this is weird. I think I know how we can set up parallel without using any foreign actions